openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -117,6 +118,8 @@ def get_object(self):
 
         # TODO: TNL-7185 Legacy: Refactor to return the expiration date and format the message in the MFE
         overview.course_expired_message = generate_course_expired_message(self.request.user, overview)
+        # TODO: TNL-XXXX Legacy: Refactor to return the offer data and format the message in the MFE
```

Comment:
I've updated https://openedx.atlassian.net/browse/TNL-7185 to also include this offer data